content/id/docs/concepts/architecture/cgroups.md

```diff
@@ -98,7 +98,7 @@ Untuk cgroup v2, outputnya adalah `cgroup2fs`.
 
 Untuk cgroup v1, outputnya adalah `tmpfs.`
 
-## {{% heading "Selanjutnya" %}}
+## Selanjutnya
```

Comment:
this should not be hardcoded, `whatsnext` is the right one, it will infer to ID language automatically
```suggestion
## {{% heading "whatsnext" %}}
```

ref: 
https://raw.githubusercontent.com/kubernetes/website/main/content/ja/docs/concepts/architecture/cgroups.md